cityofaustin/atd-vz-data
# Add Council District Areas to the VZ database

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/12701

Modified code from [Moped](https://github.com/cityofaustin/atd-moped/tree/main/moped-toolbox/council_districts_load) :p

There was [discussion](https://github.com/cityofaustin/atd-data-tech/issues/12701#issuecomment-1638511726) about wrapping this into an ETL that runs on a regular schedule so that we dont have to worry about manually updating if the council districts are updated in the AGOL endpoint. Since that wasnt resolved yet I went ahead and just did this as normal and if we want to do an ETL automation maybe that could be a future issue? We can also discuss here if that seems like something we should do! (Could also be done for the engineering areas)

## Testing
**URL to test:** 
None

**Steps to test:**
1. If you want you can test the scripts on your local VZ DB
2. You can also see the `council_districts` table in the staging db with the data populated
3. You can see a preview of the council districts in data/council_districts.geojson if you click on "View File" and compare to https://austin.maps.arcgis.com/home/item.html?id=8251869f87e24d3abff3d7a36fc7c4f4

![image](https://github.com/cityofaustin/atd-vz-data/assets/67130963/a3c0d7ec-ad39-47ba-b01e-98dc13e6bddc)

Steps for merge to prod:

- [ ] Drop current deprecated council_districts table in prod
- [ ] Run sql command to create new table
- [ ] Run the insert script for prod

---
#### Ship list
